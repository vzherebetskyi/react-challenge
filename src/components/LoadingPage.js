import React from 'react';

import { LoadingComponent } from './reusedComponents/LoadingComponent';

const LoadingPage = () => (
  <div className="loader-container">
    <LoadingComponent />
  </div>
);

export default LoadingPage;
