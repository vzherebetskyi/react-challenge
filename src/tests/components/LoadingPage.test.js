import React from 'react';
import renderer from 'react-test-renderer';

import LoadingPage from '../../components/LoadingPage';

test('should render LoadingPage correctly', () => {
  const elem = renderer.create(<LoadingPage />).toJSON();
  expect(elem).toMatchSnapshot();
});
