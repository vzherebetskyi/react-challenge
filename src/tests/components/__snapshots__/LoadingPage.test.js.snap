// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render LoadingPage correctly 1`] = `
<div
  className="loader-container"
>
  <ul
    className="loader"
  >
    <li />
    <li />
    <li />
    <li />
    <li />
    <li />
    <li />
    <li />
    <li />
  </ul>
</div>
`;
